ui/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Commit message: Please provide *brief* rationales why these technologies were used, esp. when it comes to less obvious choices like Vite (maybe) and pnpm.

Hint: For React, we could way it's already being used at least in ORT's web-app report, and for pnpm we could say that at least it's supported by ORT's analyzer (I'd not use a technology that's not supported by ORT's analyzer).

Reply:
How was this file created? If it was created by some React / Vite specific init command, please name that command in the commit message.

Reply:
Should we talk about the directory name here, is "ui" fine with everyone? Should we maybe use "frontend" or so instead?

Reply:
> esp. when it comes to less obvious choices like Vite (maybe) and pnpm.

For Vite, you might "borrow" some ideas for a rationale from [over here](https://github.com/oss-review-toolkit/ort/pull/7952/commits/a9764049400ab26cc93160358ee200739ec98e0d).